rero_ils/modules/documents/jsonschemas/documents/document_subjects_links-v0.0.1.json

```diff
@@ -0,0 +1,186 @@
+{
```

Comment:
It would be nice to have the links in the same field than the local subjects. This would be consistent with field `contribution`.